prisma/datamodel.prisma

```diff
@@ -40,3 +41,14 @@ type Booking {
     remark: String
     createdAt: DateTime! @createdAt
 }
+
+type Event {
+    id: ID! @id
+    user: User! @relation(name: "UserEvents")
```

Comment:
I would suggest use the name `organiser`.

I suggest so because in the future we might have `users` who join the event as a list.

Reply:
in that case, there should be an `organiser: User!` field, and a `participants: [User]` field

Reply:
alrightt, I'll move to `participants` field once there's no remaining flaw for the basics